.golangci.yml

```diff
@@ -20,6 +23,7 @@ linters:
     - nakedret
     - nolintlint
     - staticcheck
+    - revive
```

Comment:
What does revive?

Reply:
Revive does a lot!  

In this case it's responsible for some of the naming changes, but it's a core piece of golangci-lint.  (similar to staticcheck or stylecheck, it's general purpose)